src/templates/neutron.conf

```diff
@@ -1127,6 +1127,16 @@ password = {{ identity_service.service_password }}
 
 [nova]
 
+auth_url = {{ identity_service.internal_protocol }}://{{ identity_service.internal_host }}:{{ identity_service.internal_port }}
```

Comment:
does this block need the same conditional guard as in the neutron-operator?

Reply:
nova-operator rather!

Reply:
Done